commons/helpers/health_helper.py

```diff
@@ -269,9 +269,10 @@ def is_motr_online(self) -> bool:
                 pod_name = node["name"]
                 services = node["svcs"]
                 for service in services:
-                    if service["status"] != "started":
-                        LOG.error("%s service not started on pod %s", service["name"], pod_name)
-                        return False
+                    if service["name"] != "motr_client":
```

Comment:
do we want to have similar check in other methods or we just want to enable health check with m0client and so just need to check_node_health?

Reply:
We have already done the same thing in HA lib for cluster shutdown test cases. We are skipping the health check for motr client which is not required. This is needed in this function as this was blocking the motr tests execution.